packages/stencil/components/pagination/src/components/osds-pagination/osds-pagination.scss

```diff
@@ -0,0 +1,52 @@
+@import 'styles/osds-pagination.color';
+@import 'styles/osds-pagination.size';
+@import '~@ovhcloud/ods-theming/color/ods-theming-color';
+@import '~@ovhcloud/ods-theming/ods-theme';
+
+// CSS for the component but not for theming
+
+:host {
+  cursor: pointer;
+  display: inline-block;
+  position: relative;
+  text-align: left;
+
+  ul {
+    list-style-type: none;
+  }
+
+  li {
+    display: inline;
+    list-style-type: none; 
+    white-space: nowrap;
+  }
+
+  .ellipsis, .disabled {
+    cursor: not-allowed;
+    pointer-events: auto;
+  }
+
+}
+
+:not([disabled]):focus-visible {
+  @include osds-pagination-on-selected-host() {
+    outline-style: dotted;
+    outline-width: var(--ods-size-inset-02);
```

Comment:
i let you @astagnol sho to you what you can done better with design tokens and css variables about this here

Reply:
This is a specific size related to outline on focus components.
It needs a whole refactoring for that type of design token. #a11y